scripts/gen_vimdoc.py

```diff
@@ -96,7 +96,6 @@
             'win_config.c',
             'tabpage.c',
             'ui.c',
-            'extmark.c',
```

Comment:
Why is this removed?

Reply:
There seems to be already a same entry above.

Reply:
Zeer is correct.

I guess I could have put the fix in the typo squash PR or in a separate PR altogether but felt like it was too small a change for that.

Reply:
I'll remove that change and create a new PR for that.